geometry_msgs/Transform initial_estimate
sensor_msgs/PointCloud2 target_cloud
---
geometry_msgs/TransformStamped template_pose
string template_name
float64 match_error
---
